test.js

```diff
@@ -276,3 +277,24 @@ cmd.title = (message, expected) => `cmd is: ${JSON.stringify(expected)}`;
 test(cmd, ' foo bar', 'foo', 'bar');
 test(cmd, ' baz quz', 'baz', 'quz');
 test(cmd, '');
+
+async function spawnAndKill(t, signal) {
+	const cp = m('sub-process');
+	let pid;
+
```

Comment:
`cp.stdout.setEncoding('utf8')`

Then you don't have to `.toString()` it below.